managed/devops/bin/yb_platform_util_py2.py

```diff
@@ -0,0 +1,498 @@
+    universe_url = base_url + "/api/v1/customers/" + customer_uuid + "/universes"
+    response = call_api(universe_url, auth_uuid)
+    data = convert_unicode_json(json.load(response))
+    for universe in data:
+        if universe["name"] == universe_name:
+            del universe['pricePerHour']
+            return universe
+
+@exception_handling
+def get_universe_by_uuid(base_url, customer_uuid, auth_uuid, universe_uuid):
+    """
+    Get universe details by UUID of the universe.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param universe_uuid: UUID of the universe.
+    :return: None
+    """
+    universe_config_url = base_url + "/api/v1/customers/" + customer_uuid + \
+                          "/universes/" + universe_uuid
+    response = call_api(universe_config_url, auth_uuid)
+    return convert_unicode_json(json.load(response))
+
+
+def get_universe_uuid(base_url, customer_uuid, auth_uuid, universe_name):
+    """
+    Get the UUID of the universe.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid:  authentication token for the customer.
+    :param universe_name: Name of the universe.
+    :return: None.
+    """
+    universe = get_universe_by_name(base_url, customer_uuid, auth_uuid, universe_name)
+    if universe:
+        response = {"data": universe["universeUUID"], "status": "success", "error": ""}
+        print(response)
+    else:
+        message = "Universe with {0} not found".format(universe_name)
+        response = {"data": message, "status": "failed", "error": message}
+        print(response)
+
+
+@exception_handling
+def get_customer_uuid(base_url, auth_uuid):
+    """
+    Get customer UUID.
+
+    :param base_url: base url of the platform back end.
+    :param auth_uuid: authentication token for the customer.
+    :return: None
+    """
+    customer_url = base_url + "/api/v1/customers"
+    response = call_api(customer_url, auth_uuid)
+    data = convert_unicode_json(json.load(response))
+    if (len(data) == 1):
+        response = {"data": data[0], "status": "success", "error": ""}
+        print(response)
+    else:
+        response = {"data": data, "status": "failed", "error": "Please provide customer UUID"}
+        print(response)
+
+
+@exception_handling
+def delete_universe_by_id(base_url, customer_uuid, auth_uuid, universe_uuid):
+    """
+    Delete the universe by providing UUID.
+
+    :param base_url: base url of the platform back end.
+    :param customer_uuid: UUID of the customer.
+    :param auth_uuid: authentication token for the customer.
+    :param universe_uuid: UUID of the universe to be deleted.
+    :return:
+    """
+    universe_delete_url = base_url + "/api/v1/customers/" + customer_uuid + "/universes/" \
+                          + universe_uuid + "?isForceDelete=true"
```

Comment:
Are we intentionally doing Forced delete? I don't like this. Add one more key to the script which forces the universe deletion. Without the key don't use forced delete.

Reply:
yes. For the initial version, we thought to do force delete. If you having concern let me discuss with Arnav again.